lib/data/wrapper.js

```diff
@@ -305,6 +305,62 @@ const data = {
         return client.getDiskUsage(log.getSerializedUids(), cb);
     },
 
+    _dataCopyPut: (serverSideEncryption, stream, part, dataStoreContext,
+    destBackendInfo, log, cb) => {
+        if (serverSideEncryption) {
+            return kms.createCipherBundle(
+            serverSideEncryption,
+            log, (err, cipherBundle) => {
+                if (err) {
+                    log.debug('error getting cipherBundle');
+                    return cb(errors.InternalError);
+                }
+                return data.put(cipherBundle, stream,
```

Comment:
minor: this and the `data.put` call below look the same except you are not passing a cipherBundle, so you might be able to refactor them into a common function / callback, or use async to reduce the lines of code. Up to you if it's worth it though